modules/ipc/src/zenoh/conversions.cpp

```diff
@@ -0,0 +1,70 @@
+//=================================================================================================
+// Copyright (C) 2023-2024 HEPHAESTUS Contributors
+//=================================================================================================
+
+#include "hephaestus/ipc/zenoh/conversions.h"
+
+#include <chrono>
+#include <cstddef>
+#include <cstdint>
+#include <numeric>
+#include <span>
+#include <string>
+#include <string_view>
+#include <vector>
+
+#include <fmt/core.h>
+#include <zenoh/api/bytes.hxx>
+#include <zenoh/api/id.hxx>
+#include <zenoh/api/timestamp.hxx>
+
+namespace heph::ipc::zenoh {
+namespace {
+[[nodiscard]] auto toChrono(uint64_t timestamp) -> std::chrono::nanoseconds {
```

Comment:
Add units to timestamp, or are these `nanoseconds`?

Reply:
it is not nanoseconds, it is a weird format described in the documentation linked below

Reply:
merci